web/src/HUD.tsx

```diff
@@ -89,7 +89,11 @@ class HUD extends Component<HudProps, HudState> {
   componentWillMount() {
     incr("ui.web.init", { ua: window.navigator.userAgent })
     this.unlisten = this.history.listen((location, _) => {
-      let tags = { path: location.pathname }
+      let isLog =
```

Comment:
can you add of mappings from pathname -> tags? this feels very brittle

Reply:
Yeah let me merge this in now but I'll make myself a ticket to do that tomorrow.